### PR TITLE
Hide the traffic breakdown if we cannot display stats

### PR DESCRIPTION
**Pull request by jjolmo**
Related to #

## Proposed Changes

- If stats are disabled (when organic_views is -1) then we just display this instead in "traffic breakdown"
 
image
 

## Testing Instructions
- Promote a campaign in jetpack site and disable their stats
- Try to open the new created campaign. You should see this new message  (as described in the image)

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by sbarbosa — APPROVED** [review]
LGTM 👍

**Comment by jjolmo** [comment]
It seems I found a weird condition where we were not displaying stats if there wasn't at least one click registered since the campaign was run. I'll fix this in the DSP before merging this PR

**Comment by jjolmo** [comment]
Done, the condition is fixed in the DSP and I've updated this PR to match the new changes

File: client/data/promote-post/use-promote-post-campaigns-query-new.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export type CampaignResponse = {
 		total_budget_used: number;
 		display_delivery_estimate: string;
 		views_total: number;
+		stats_enabled: number;
```
Comment (sbarbosa) [inline client/data/promote-post/use-promote-post-campaigns-query-new.ts]: I think we return this as boolean in the DSP.

**Review by sbarbosa — COMMENTED** [review]
I tested it again but I am not seeing the `stats_enabled` attribute in the response. 
This is what I am getting for my atomic and simples sites:
```
        "campaign_stats": {
            "impressions_total": 0,
            "clicks_total": 0,
            "clickthrough_rate": 0,
            "duration_days": 0,
            "budget_day": 5,
            "spent_budget_cents": 0,
            "budget_left": 0,
            "total_budget": 0,
            "total_budget_used": 0,
            "views_organic": null,
            "views_organic_rate": 0,
            "views_ad_rate": 0
        },
```

I checked and the DSP PR is merged, its strange that I am not getting that new attribute.

**Review by timur987 — APPROVED** [review]
Thanks, @jjolmo!

I looked through the code and found it quite good (though I agree with Sebastián's comment).

Unfortunately, I can't create a campaign (using a TeamPay virtual credit card) via local Jetpack site, so I can't test how it actually works.

File: client/data/promote-post/use-promote-post-campaigns-query-new.ts (hunk 1 above)
Comment (jjolmo) [reply client/data/promote-post/use-promote-post-campaigns-query-new.ts]: whoops, changed

**Comment by jjolmo** [comment]
> I checked and the DSP PR is merged, its strange that I am not getting that new attribute.

It appears when fetching data of a campaign. This change is now live in production
image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8063216

Thank you @jjolmo for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8063216).

**Pull request merged**